src/Native/System.Native/pal_runtimeinformation.h

```diff
@@ -8,11 +8,14 @@
 
 extern "C" int32_t SystemNative_GetUnixVersion(char* version, int* capacity);
 
-extern "C" int32_t SystemNative_GetUnixArchitecture();
+extern "C" int32_t SystemNative_GetOSArchitecture();
```

Comment:
@stephentoub Do we need to keep the old method for some time to avoid breaking stuff?